test/rulesets/Base/indexing.jl

```diff
@@ -77,6 +77,91 @@
             test_rrule(getindex, x, [2,2], [3,3])
         end
     end
+    
+    @testset "getindex for structured arrays" begin
+        test_frule(getindex, Diagonal(rand(3)), 1)
+        test_frule(getindex, Symmetric(rand(3, 3)), 2, 3)
+        
+        test_rrule(getindex, Diagonal(rand(3)), 1)
+        @test_skip test_rrule(getindex, Diagonal(rand(3)), 2, :)  # in-place update of off-diagonal entries
```

Comment:
Comment on why we are skipping this one?
And I suspect a link to a CRTU issue?

Reply:
Now https://github.com/JuliaDiff/ChainRulesTestUtils.jl/issues/260